### PR TITLE
Explore Page (index.md)

### PR DESCRIPTION
**Pull request by IMB11**

### CODE REVIEW

File: .dev/view.html [hunk 1]
```diff
@@ -8,8 +8,7 @@
     const client = new XMLHttpRequest();
     client.open('GET', file);
     client.onreadystatechange = function() {
-        const html = new showdown.Converter().makeHtml(client.responseText);
-        document.getElementById("preview").innerHTML = html;
+        document.getElementById("preview").innerHTML = new showdown.Converter().makeHtml(client.responseText);
```
Comment (IMB11) [inline .dev/view.html:11]: this shouldn't effect any other branches.

**Review by StormPacer — APPROVED** [review]
Looking at the layout, I approve the changes.

**Comment by IMB11** [comment]
oops

**Review by no-u101 — APPROVED** [review]
not sure why this has not been merged yet

**Pull request merged**